src/main/java/duke/DatesAndTimesFormatter.java

```diff
@@ -0,0 +1,35 @@
+package duke;
+
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Enum class for the different date and time formats.
+ */
+//@@ author zhanyang01-reused\
+//Inspired by the smart usage of enums for date and time formats.
+public enum DatesAndTimesFormatter {
+    INPUT_FORMAT("yyyy-MM-dd"),
+    OUTPUT_FORMAT("MMM dd yyyy"),
+    SLASH_FORMAT("dd/MM/yyyy");
```

Comment:
add more formats to make user input foolproof

Reply:
I agree. I shall add more formats to handle user input.